geniza/corpus/templates/corpus/snippets/document_transcription.html

```diff
@@ -13,6 +13,25 @@
     {# translators: label for checkbox toggle to show the translation panel for a document #}
     <input type="checkbox" id="translation-on" aria-label="{% translate 'show translation' %}" disabled="true"/>
     <label for="translation-on"><svg><use xlink:href="{% static 'img/ui/all/all/translation-toggle.svg' %}#translation-toggle" /></svg></label>
+    <div class="dropdown-container">
```

Comment:
I think you're making the layout and transitions harder than necessary. Also, we're going to need something similar to this for the translations panel eventually, when we add support for that, so the more we can use the existing panel layout & grid, the better.

I suggest you move this inside the `panel-container` grid element and have a first row of control panel/label elements: an empty image div, and this drop-down/label for transcriptions.

Reply:
One question about this: if we put the dropdown inside the `panel-container` grid, it may not match the designs. In the designs, the image or transcription panel can sometimes differ from the width/position of its own toggle, whereas the dropdown is always aligned exactly with the toggle. Maybe this is a question for Gissoo, but is that the desired effect? Or should we always have the panels _and_ the dropdown match the toggle width? 

If I move the dropdown inside the grid, it's going to match the panel width, so it will definitely be misaligned from the toggles as it is now:

![Screen Shot 2022-06-21 at 2 27 25 PM](https://user-images.githubusercontent.com/7234006/174873804-e4935082-f60f-4622-9f84-e992c6620346.png)

![Screen Shot 2022-06-21 at 2 27 10 PM](https://user-images.githubusercontent.com/7234006/174873789-a8b88372-225a-4a9d-86d8-9a824c5832d5.png)


We could adjust the panels themselves to always match the toggles, but that will differ from the designs. You can see them unaligned here in Gissoo's design:

![Screen Shot 2022-06-21 at 2 33 00 PM](https://user-images.githubusercontent.com/7234006/174872985-1e59e882-3022-4e9d-95fd-8bf8f24276e1.png)

I also fear that this could create a problem on smaller screens where the transcription will become too narrow:

![Screen Shot 2022-06-21 at 2 34 33 PM](https://user-images.githubusercontent.com/7234006/174873352-e3f76c64-b6ff-4e31-80f6-35cd9c958b74.png)


So I guess to summarize, my question is: what should be aligned with what here? And if the alignments and spacing in my current work on the branch are correct (they match the design currently), then should we prioritize that over the grid reuse issue with the code (or is there another solution)?

Reply:
@rlskoeser They are not! Only the dropdowns are aligned with the toggles now. Take a look at Gissoo's design in the third image I posted just now, and you can see how the image panel extends past its toggle, and the transcription panel is narrower than its toggle and shifted slightly to the right. So in my implementation, I tried to match the design where the dropdown is always aligned to the toggle, but the panels may not be (and usually aren't).

With the "transcription panel being too narrow" comment, I meant to say that would be a possible side-effect if we aligned the panels with the toggles exactly. Though it would likely be alleviated if we ignore the unselected translation toggle on the side—we would just need to rethink some margins and proportions. Generally it would need us to pass the designs/margins back to Gissoo, so that she can design everything with that alignment (toggles and panels aligned together) in mind.